2_delivery_care.do

```diff
@@ -50,7 +50,11 @@ c_sba_eff2_q
 		if inlist(country_name, "Georgia2018") {
 			replace c_hospdel = 0 if mn20 != .
 			replace c_hospdel = 1 if mn20 == 41		// 1 for private/public hospital
-		}	
+		}
+		if inlist(country_name,"Tonga2019") {
+			replace c_hospdel = 0 if mn20 != .
+			replace c_hospdel = 1 if mn20 == 21 	// 1 for public hospital
```

Comment:
@Ortsang On treatment of hospitals, do we see any mention of private hospitals in the tabulation for MN20?

Reply:
no, mn20 only has observations from "PRIVATE MATERNITY HOME" and "OTHER"

Reply:
OK, this is bizarre and helpful to note. Typically we see across surveys that private hospitals are listed...